components/brave_wallet_ui/components/desktop/views/portfolio/components/token-lists/token-list.style.ts

```diff
@@ -5,6 +5,49 @@
 
 import styled from 'styled-components'
 
+// Constants
+import {
+  layoutPanelWidth
+} from '../../../../wallet-page-wrapper/wallet-page-wrapper.style'
+
+// Shared Styles
+import {
+  Row
+} from '../../../../../shared/style'
+
 export const HorizontallyPaddedDiv = styled.div`
   padding-right: 12px;
 `
+
+export const SearchBarWrapper = styled(Row) <{
+  showSearchBar: boolean
+}>`
+  width: 230px;
+  @media screen and (max-width: ${layoutPanelWidth}px) {
+    display: ${(p) => p.showSearchBar ? 'flex' : 'none'};
+    width: 100%;
+  }
+`
+
+export const ControlBarWrapper = styled(Row) <{
+  showSearchBar: boolean
+  isNFTView?: boolean
+}>`
+  padding: 0px 32px;
+  margin-bottom: 16px;
+  @media screen and (max-width: ${layoutPanelWidth}px) {
+    padding: ${(p) => p.showSearchBar
+    ? p.isNFTView
+      ? '2px'
+      : '0px'
+    : '4px'} 24px 0px 24px;
+    margin-bottom: ${(p) => p.showSearchBar ? 12 : 16}px;
+  }
+`
+
+export const SearchButtonWrapper = styled(Row)`
+  display: none;
+  @media screen and (max-width: ${layoutPanelWidth}px) {
+    display: flex;
+  }
+`
```

Comment:
nit: define these styles elsewhere, since these styles are being consumed by more than just `token-list`

Reply:
Moved to `Portfolio` style.ts.`